pkgs/development/libraries/immer/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
```

Comment:
Is there any C compilation happening? Or it is just a set of headers?

If gcc is not needed, then you can change this to `stdenvNoCC`

Reply:
I believe there is no compilation happening, but the headers are installed with `cmake` and the project's `CMakeLists.txt` configuration requires cpp compiler to be present. I can try patching `CMakeLists.txt`, but I think it is simpler to just use `stdenv` here

Reply:
Don't worry.
I asked because you set `dontBuild = true;` and it is typical in packages that do not need CC or CXX.
But since the code needs it somehow, just ignore what I said.